views/newsletters.html

```diff
@@ -137,6 +137,7 @@
     'selectTemplateTitle': __('Select a responsive template'),
 
     'draftNewsletterTitle': __('Subject'),
+    'draftPostNotificationTitle': __('The last [newsletter:number] posts from our blog'),
```

Comment:
If you meant to include the number of posts contained in the current post notification, the shortcode should be `[newsletter:total]`. The `[newsletter:number]` shortcode displays an issue number (how many post notifications of this kind were sent + 1).